### PR TITLE
Hdm 399 file size ltsi range

### PR DESCRIPTION
**Pull request by dlpierce**
Adds file_size properties to the model ingest, and installs the blacklight_range_limit gem.

### CODE REVIEW

File: app/indexers/file_set_indexer.rb [hunk 1]
```diff
@@ -1,8 +1,13 @@
 class FileSetIndexer < CurationConcerns::FileSetIndexer
+
   def generate_solr_document
     super.tap do |solr_doc|
       solr_doc[Solrizer.solr_name('filename')] = object.filename
 
+      solr_doc.delete Solrizer.solr_name(:file_size, STORED_INTEGER)
+      solr_doc[Solrizer.solr_name(:file_size, Solrizer::Descriptor.new(:long, :stored, :indexed))] = object.file_size[0].to_i / 1000000
```
Comment (afred) [inline app/indexers/file_set_indexer.rb]: @dlpierce I can see my future self not remembering this is in MB. Opposed to naming it `:file_size_megabytes`, or `:file_size_mb`?

**Comment by afred** [comment]
I'll rebase this.. stand by..

**Comment by afred** [comment]
Also, this should be merged into `master`, yes?

File: app/indexers/file_set_indexer.rb (hunk 1 above)
Comment (dlpierce) [inline app/indexers/file_set_indexer.rb]: :file_size_mb is what I originally used. So that is fine with me

**Comment by amolmkhedkar** [comment]
I think some parts could go into master.

**Pull request merged**